ownCloud/Bookmarks/BookmarkViewController.swift

```diff
@@ -727,6 +743,21 @@ class BookmarkViewController: StaticTableViewController {
 	func isAuthenticationMethodTokenBased(_ authenticationMethodIdentifier: OCAuthenticationMethodIdentifier) -> Bool {
 		return authenticationMethodTypeForIdentifier(authenticationMethodIdentifier) == OCAuthenticationMethodType.token
 	}
+	
+	// MARK: - Keyboard AccessoryView
+	@objc func toogleTextField (_ sender: UIBarButtonItem) {
```

Comment:
@hosy Instead of comparing `sender.style` I would suggest to create separate action methods

Reply:
Make sense in this case